src/client/components/componentStyle.css

```diff
@@ -3,51 +3,114 @@
     padding: 10px;
```

Comment:
I tend to not ever review the actual CSS used, if it looks good the CSS is usually good. 
What I do look at is the names of classes, like are you writing btn instead of button.
Or are you naming classes contentDiv because it's added to a <div> or naming it flexContainer simply because the class uses flex. 

CSS classes should describe what a part of HTML is, so writing submitButton is fine. It's not because it uses <button> but because it looks like a button to the user! And when they click it, it submis the form.

Other than that, I would avoid "joined selector" styles, like "submitButton ul", instead name what that <ul> means in the context of the submitButton and create a dedicated class for that and add it to the ul directly.